vars/savePodsLogs.groovy

```diff
@@ -0,0 +1,20 @@
+import uk.gov.hmcts.contino.DockerImage
+
+
+def call(DockerImage dockerImage, Map params, String scope) {
+  onPR {
```

Comment:
The core of the problem I am trying to solve is on the PRs, hence I tried to limit my changes to the PRs.

I thought about master, but then I thought that since master deployments coincide with production deployments by saving the logs there too I would have potentially risked disclosing data to users that are not meant to see it (the developers).

We're always in time to remove this afterwards should it be deemed necessary in production as well.

Reply:
Makes sense and I learned something new. I'll remove that onPR then.